### PR TITLE
PET-16 Go coverage only in cmd and pkg directories

### PR DESCRIPTION
**Pull request by glothriel**
New go version, parametrized.

### CODE REVIEW

File: go/test/action.yaml [hunk 1]
```diff
@@ -54,7 +58,7 @@ runs:
 
     - name: Run tests
       shell: bash
-      run: go test ./... -v -coverpkg=./... -coverprofile cover.out
+      run: go test ./cmd/... ./pkg/... -v -coverpkg=./cmd/...,./pkg/... -coverprofile cover.out
```
Comment (radekska) [inline go/test/action.yaml:61]: What if some modules would have an [internal](https://github.com/golang-standards/project-layout#internal) directory as well?

Comment (Decard6) [reply go/test/action.yaml:61]: What purpose does this change have?

Comment (glothriel) [reply go/test/action.yaml:61]: @Decard6 https://github.com/PiwikPRO/PPMS-DomainResolver/pull/60/files
@radekska I think that we should introduce a global convention of putting the go project files only in `pkg` and `cmd` directory in order for it to work. If you plan to use `internal` packages, there's no problem in putting them under `pkg/internal` i suppose.

Comment (glothriel) [reply go/test/action.yaml:61]: @radekska Hmm, now i've read your link. Should we add the `internal` package to the flags as well?

Comment (radekska) [reply go/test/action.yaml:61]: @glothriel not sure how many modules actually use a separate `internal` package, though making test and coverage directories configurable would be a plus

Comment (Decard6) [reply go/test/action.yaml:61]: I'd say so, we probably want to adjust to `go` standards.

Comment (glothriel) [reply go/test/action.yaml:61]: I'm going to get back to this on next pet project friday

**Pull request closed without merging**